Sources/MapboxNavigation/OrnamentsController.swift

```diff
@@ -296,6 +349,21 @@ extension NavigationMapView {
             })
         }
         
+        private func calculateJaccardSimilarity(targetString: String, compareSet: Set<String>, allCounts: Int) -> Double {
```

Comment:
Let's move this method into `Set` extension, there is a chance that it might be useful in future. Instead of passing raw `targetString` I think it'd make sense to use `Set` directly. Based on proposed algorithm below `allCounts` will not be needed.

Reply:
Thanks for the suggestion, due to the following comment, the function is moved to the `String` extension.